pkgdb2/templates/package_request.html

```diff
@@ -44,12 +43,20 @@
             document.body.style.cursor = 'wait';
             $('#pkgname').val('');
             $('#summary').val('');
+            $('#pkgname').prop('disabled', true);
```

Comment:
If you're disabling pkgname and summary, should you disable/reenable the review url field too?